oddt/fingerprints.py

```diff
@@ -214,7 +214,7 @@ def fold(fp, size):
 def sparse_to_dense(fp, size, count_bits=True):
     """Converts sparse fingerprint (consists only 'on' bits)
     to dense (consists all bits)"""
-    sparsed_fp = np.zeros(size, dtype=np.uint8 if count_bits else bool)
+    sparsed_fp = np.zeros(size, dtype=np.uint32 if count_bits else bool)
```

Comment:
`np.uint8` stores int up to 255. As I recall corectly in PDBbind v2016 the highest occurring bit had this set to ~80. This needlessly increases the FP size by 4. We maybe drive that through `np.unique` with `return_counts` and infer dtybe by max count instead of `np.add.at`. But this approach would need to be properly benchmarked should it introduce performance regression.

I'm against setting it to to fixed `np.uint32`.